pkg/cmd/roachtest/spec/machine_type.go

```diff
@@ -12,13 +12,16 @@ package spec
 
 import "fmt"
 
-// AWSMachineType selects a machine type given the desired number of CPUs.
-func AWSMachineType(cpus int, highmem bool) string {
+// AWSMachineType selects a machine type given the desired number of CPUs and
+// memory per CPU ratio.
+func AWSMachineType(cpus int, mem MemPerCPU) string {
 	// TODO(erikgrinaker): These have significantly less RAM than
 	// their GCE counterparts. Consider harmonizing them.
 	family := "c5d" // 2 GB RAM per CPU
-	if highmem {
+	if mem == High {
 		family = "m5d" // 4 GB RAM per CPU
+	} else if mem == Low {
+		panic("low memory per CPU not implemented for AWS")
```

Comment:
Nit: I would probably phrase it as `not available` instead of `not implemented`. AWS doesn't seem to have instance types with mem/vcpu ratio matching GCE's `highcpu` instances.